data/hai/unfettered side missions.txt

```diff
@@ -0,0 +1,531 @@
+			`	"Use the Ladybug as your flagship.`
+			`	"And do not bring any other ships."`
+			`	Given the dangers of Unfettered space, it would be a good idea to familiarize yourself with the design and even customize it more to your own liking. Especially if you do plan to deliver this cargo, as requested, in only this single ship.`
+				accept
+	on enter
+		system
+			attributes "unfettered"
+		conversation
+			branch "not moded bug"
+				not "flagship model: Ancient Ladybug"
+			branch "not single ship"
+				"total ships" != 1
+			action
+				"old reputation: Hai (Unfettered)" = "reputation: Hai (Unfettered)"
+				"reputation: Hai (Unfettered)" >?= 0
+			`As you enter the system, you prepare for a fight to get the cargo through, but, surprisingly, it is very quiet this time.`
+			`	Indeed, for the first time, entering Unfettered space does not mean getting shot at by missiles from all sides.`
+			`	They are not hailing you either. Perhaps they think you are one of the Hai aiding them, especially given your cargo and the ship you are flying.`
+				accept
+			label "not single ship"
+			`You enter the system, and, for a brief moment, nobody is shooting you.`
+			`	The Unfettered quickly change their mind, however, as soon as your escorts enter the system, and they happily head your way, weapons ready, most probably with the intention of looting your cargo.`
+			`	It becomes clear you should have come with only the Ancient Ladybug, as the associates suggested.`
+				decline
+			label "not moded bug"
+			`You enter the system and the Unfettered shoot you? as usual. Perhaps they would have acted differently if you had used the Ancient Ladybug as a flagship, but it is too late to find out now.`
+				decline
+	on fail
+		"reputation: Hai (Unfettered)" <?= -1000
+		dialog `The Unfettered and their associates are clearly not happy when they somehow find out you did not get the cargo there in time. The message they send you is short: "You will have to prove yourself another way."`
+	on visit
+		dialog `You have reached <planet>, but you either did not bring the Ancient Ladybug, some of the cargo was on another ship, or you are not using the Ladybug as your flagship. You should park your fleet and only use the Ancient Ladybug.`
+	to complete
+		has "flagship model: Ancient Ladybug"
+	on complete
+		dialog `You receive a message from the 'associates of Hai-home': "Thank you for helping our brethren, <last>. We will not forget this. Please accept these <payment> to get you started on paying the gift you still need to prove yourself worthy of."`
+		payment 1000000
+		log "Delivered a pressing food cargo to the Unfettered using a newly acquired but rather old Ladybug. They seem to not be firing on me, for now at least."
+
+
+
+# no category, meaning you can't remove it from the Ladybug
+outfit "Unfettered Landing Permit"
+	"maintenance costs" 681
```

Comment:
it's like this on purpose because it's 600 like she said + taxes of 13.5%

Reply:
Sure, but doesn't match the rest of the game. Taxes exist everywhere else too, right? All the ship prices, outfit prices, and other salaries aren't exact numbers like this one. For gameplay's sake, this should still be rounded. There's a distinct advantage to having more rounded numbers because they're easier for the player to run quick math with. I'm not aware of any advantages that using a number this exact provides.

Reply:
alright 650 it is